services/storage/artifacts.mk

```diff
@@ -0,0 +1,15 @@
+# Create new tls certs
+
+STORAGE_DIR=$(patsubst %/,%,$(dir $(abspath $(lastword $(MAKEFILE_LIST)))))
+
+get.storage:
+	@echo "⇒ Creating tls certs to NeoFS node"
+	@(echo "[req]"; \
+      echo "distinguished_name=req"; \
+      echo "req_extensions=san"; \
+      echo "[san]"; \
+      echo "subjectAltName=DNS:s04.${LOCAL_DOMAIN}") > tmpconf
+	@openssl req -new -newkey rsa:4096 -x509 -sha256 -days 3650 -nodes \
```

Comment:
I think it is time to add `Prerequisites` section in readme. It should include:
- docker,
- docker-compose,
- make,
- expect,
- openssl,
- ???